src/flows/context/schemaProvider.tsx

```diff
@@ -42,17 +42,78 @@ export const SchemaContext = React.createContext<SchemaContextType>(
   DEFAULT_CONTEXT
 )
 
+const csvToSchema = (result: RunQueryResult): unknown => {
+  let ni, no
+  const filtered = [
+    /^_start$/,
+    /^_stop$/,
+    /^_time$/,
+    /^_value/,
+    /^_measurement$/,
+    /^_field$/,
+    /^table$/,
+    /^result$/,
+  ]
+  if (!result) {
+    return
+  }
+  if (result?.type !== 'SUCCESS') {
+    throw new Error(result.message)
+  }
+
+  const out = fromFlux(result.csv).table as any
```

Comment:
shouldn't the result already have a parsed result?

Reply:
Nah, currently we're just setting the value that's returned from the query